src/DssSpell.sol

```diff
@@ -42,86 +90,62 @@ contract DssSpellAction is DssAction {
-
-    uint256 internal constant MILLION = 10 ** 6;
-    // uint256 internal constant RAY  = 10 ** 27;
-    uint256 internal constant WAD     = 10 ** 18;
-
-    ChainLogLike internal immutable CHAINLOG    = ChainLogLike(DssExecLib.getChangelogAddress("CHANGELOG"));
-
-    address internal immutable FLASH_KILLER     = DssExecLib.getChangelogAddress("FLASH_KILLER");
-    address internal immutable MCD_FLASH        = DssExecLib.getChangelogAddress("MCD_FLASH");
-    address internal immutable MCD_FLASH_LEGACY = DssExecLib.getChangelogAddress("MCD_FLASH_LEGACY");
 
-    address internal immutable MCD_PSM_PAX_A    = DssExecLib.getChangelogAddress("MCD_PSM_PAX_A");
-    address internal immutable MCD_PSM_GUSD_A   = DssExecLib.getChangelogAddress("MCD_PSM_GUSD_A");
+    // uint256 internal constant MILLION = 10 ** 6;
+    // uint256 internal constant RAY     = 10 ** 27;
+    // uint256 internal constant WAD     = 10 ** 18;
 
     function actions() public override {
+        // ------------------ Pause Optimism Goerli L2DaiTeleportGateway -----------------
+        // Forum: https://forum.makerdao.com/t/community-notice-pecu-to-redeploy-teleport-l2-gateways/19550
+        // L2 Spell to execute via OPTIMISM_GOV_RELAY:
+        // https://goerli-optimism.etherscan.io/address/0xC077Eb64285b40C86B40769e99Eb1E61d682a6B4#code
+        OptimismGovRelayLike(OPTIMISM_GOV_RELAY).relay(
+            OPTIMISM_L2_SPELL,
+            abi.encodeWithSignature("execute()"),
+            OPT_MAX_GAS
+        );
+
+        // ------------------ Pause Arbitrum Goerli L2DaiTeleportGateway -----------------
+        // Forum: https://forum.makerdao.com/t/community-notice-pecu-to-redeploy-teleport-l2-gateways/19550
+        // L2 Spell to execute via ARBITRUM_GOV_RELAY:
+        // https://goerli.arbiscan.io/address/0x11dc6ed4c08da38b36709a6c8dbaac0eaedd48ca#code
+        // Note: ARBITRUM_GOV_RELAY must have been pre-funded with at least ARB_L1_CALL_VALUE worth of Ether
+        ArbitrumGovRelayLike(ARBITRUM_GOV_RELAY).relay(
+            ARBITRUM_L2_SPELL,
+            abi.encodeWithSignature("execute()"),
+            ARB_L1_CALL_VALUE,
+            ARB_MAX_GAS,
+            ARB_GAS_PRICE_BID,
+            ARB_MAX_SUBMISSION_COST
+        );
+
+        // ------------------ Pause Starknet Goerli L2DaiTeleportGateway -----------------
+        // Forum: https://forum.makerdao.com/t/community-notice-pecu-to-redeploy-teleport-l2-gateways/19550
+        // L2 Spell to execute via STARKNET_GOV_RELAY:
+        // src: https://github.com/makerdao/starknet-spells-goerli/blob/b7ca995cf1d266aa2382d85e35a86b4fae52aa15/src/spell.cairo
+        // contract: https://testnet.starkscan.co/class/0x00a052591661d7e249b46a1084c63b14dae6aa8b1a56ab3f7df8c8add1c374b1#overview
+        StarknetGovRelayLike(STARKNET_GOV_RELAY).relay{value: STA_L1_CALL_VALUE}(STARKNET_L2_SPELL);
+
+        // disallow legacy bridge on escrow
```

Comment:
do we have a link for this where Starknet published the reason/existence of this change?

Reply:
https://forum.makerdao.com/t/starknet-changes-for-executive-spell-on-the-week-of-2023-01-30/19607

Reply:
ah I see it's already added.

Reply:
added that link as a suggestion